src/wavm/PlatformThreadPool.cpp

```diff
@@ -0,0 +1,80 @@
+#include "PlatformThreadPool.h"
+
+#include <wavm/openmp/ThreadState.h>
+#include <wavm/openmp/openmp.h>
```

Comment:
Would be nice to make this non-OMP specific if possible, but probably unavoidable atm/ not worth it, so just rename as per other comments.

Reply:
Making the ThreadPool non OMP specific is too much of a game of chicken and eggs, the maze of headers is already very complex, I try to have everything in `wavm/openmp` not known about `wavm` to avoid cycles. But the `PlatformThreadPool` being OpenMP specific, I think it clearly belongs there indeed. I don't see a pretty way to do this though, I could move the `PlatformThreadPool` files inside the `openmp` folder, but build it along the `wavm` folder, and not the `wavm/openmp` lib. I'd find this confusing.

Reply:
I placed it in the openmp namespace actually, a lot more explicit probably